sphinx-docs/.screenshot_dump/.gitignore

```diff
@@ -1,3 +0,0 @@
-# Screenshots are dumped here by the build process
-*
```

Comment:
yes, but thanks for the method, I used to wonder how to force git to include empty dirs